src/app/parcels/ActionBar/ActionModals/DriverOverviewModal.tsx

```diff
@@ -32,7 +32,7 @@ const DriverOverviewInput: React.FC<DriverOverviewInputProps> = ({
     return (
         <>
             <Heading>Delivery Information</Heading>
-            <FreeFormTextInput onChange={onDriverNameChange} label="Driver's Name" />
+            <FreeFormTextInput onChange={onDriverNameChange} label="Driver's Name (required)" />
```

Comment:
I know Mina asked for the (required) bit, but seeing as shipping labels modal doesn't have this, I'm wondering if we should be consistent so either:

1. no (required) in either modal
2. (required) in both modals

personally I would lean towards 1 because less text and I think it's obvious enough from the disabling/enabling of the download button. But I'm also supportive of 2 if you think it's right @minalui @maxjohns0n  what are your thoughts? i'm not strongly fussed but I think we should be consistent

Reply:
Good point - I feel that because there are two inputs in the driver overview modal its better to make it explicit. Happy for the change to be applied to the shipping labels modal.

Reply:
Cool sounds good, @maxjohns0n please also add the (required) text to the shipping labels modal

Reply:
Done! I agree it seems better to be explicit and consistent :)